global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "5.0.301"
+        "version": "5.0.203"
```

Comment:
>203 [](http://example.com/codeflow?start=24&length=3)

can you merge latest master into this branch?

Reply:
Solved!